mathics/builtin/comparison.py

```diff
@@ -25,15 +25,13 @@
     Integer1,
     Number,
 )
-from mathics.core.symbols import Symbol
+from mathics.core.symbols import Symbol, SymbolFalse, SymbolTrue
 from mathics.core.systemsymbols import (
-    SymbolFalse,
-    SymbolTrue,
     SymbolDirectedInfinity,
     SymbolInfinity,
     SymbolComplexInfinity,
 )
-from mathics.core.numbers import dps
```

Comment:
`numbers` was rename to `number` because of a possible numpy conflict. See https://stackoverflow.com/questions/53668779/attributeerror-module-numbers-has-no-attribute-integral

After this change I was for example able to run `python mathics-core/core/atoms.py` without error.

Reply:
excelent, thanks!